client/app/lib/components/redux-form/MultiSelect.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChipInput from 'material-ui-chip-input';
+import createComponent from './createComponent';
+
+export const optionShape = PropTypes.shape({
+  id: PropTypes.number,
+  text: PropTypes.string,
+});
+
+const propTypes = {
+  onChange: PropTypes.func.isRequired,
+  label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
+  value: PropTypes.arrayOf(optionShape),
+  options: PropTypes.arrayOf(optionShape),
+  error: PropTypes.string,
+  isLoading: PropTypes.bool,
+};
+
+const styles = {
+  menuStyle: {
+    maxHeight: '80vh',
+    overflowY: 'scroll',
+  },
+};
+
+const MultiSelect = (props) => {
+  const { label, value, options, error, isLoading, onChange } = props;
+
+  return (
+    <React.Fragment>
+      <ChipInput
+        value={value || []}
```

Comment:
Is there a better way to specify the default value of `value` (ie. `[]`)?

Reply:
Ok I think can ignore this since it was updated in the subsequent commit.

Reply:
It's still there though, i can try specifying defaultProps and see if it works